requirements.txt

```diff
@@ -35,8 +35,8 @@ requests==2.26.0
 six==1.16.0
 text-unidecode==1.3
 toml==0.10.2
-torch==1.9.0
-torchvision==0.10.0
+torch==1.11.0
```

Comment:
We recently made the requirements for torch and related packages more lax by only requiring that the package version be greater than or equal to some version. I think we should keep that.

Reply:
[4a91923](https://github.com/SforAiDl/vformer/pull/88/commits/4a9192385762f003d1644d3d06d9836eaa2531df)

Reply:
@alvanli We can keep the versions for torch as 1.9.0 and torchvision as 0.10.0, right? No need to bump them.

Reply:
@alvanli Just to confirm, is `StochasticDepth` only available from version 0.11.0 onwards in torchvision?